it/src/main/java/com/google/cloud/teleport/it/datastream/DefaultDatastreamResourceManager.java

```diff
@@ -0,0 +1,306 @@
+            GcsDestinationConfig.Builder gcsDestConfigBuilder = GcsDestinationConfig
+                    .newBuilder().setPath(this.gcsRootPath);
+            if (this.destinationOutputFormat == AVRO_FILE_FORMAT) {
+                gcsDestConfigBuilder.setAvroFileFormat(AvroFileFormat.getDefaultInstance());
+            } else {
+                gcsDestConfigBuilder.setJsonFileFormat(JsonFileFormat.getDefaultInstance());
+            }
+            destinationConfigBuilder.setGcsDestinationConfig(gcsDestConfigBuilder);
+        }
+        // TODO: BQ Destination Builder
+
+        return destinationConfigBuilder.build();
+    }
+
+    @Override
+    public Stream createStream(String streamId,
+                               String sourceConnectionProfileId,
+                               String destinationConnectionProfileId)
+            throws ExecutionException, InterruptedException {
+
+        CreateStreamRequest request =
+                CreateStreamRequest.newBuilder()
+                        .setParent(LocationName.format(projectId, location))
+                        .setStreamId(streamId)
+                        .setStream(Stream
+                                .newBuilder()
+                                .setSourceConfig(buildSourceConfig(sourceConnectionProfileId))
+                                .setDisplayName(streamId)
+                                .setDestinationConfig(buildDestinationConfig(destinationConnectionProfileId))
+                                .setBackfillAll(Stream.BackfillAllStrategy.getDefaultInstance())
+                                .build())
+                        .build();
+
+        Stream reference = datastreamClient.createStreamAsync(request).get();
+        createdStreamIds.add(streamId);
+
+        return reference;
+    }
+
+    public Stream updateStreamState(String streamId, Stream.State state)
+            throws ExecutionException, InterruptedException {
+        Stream.Builder streamBuilder = Stream.newBuilder()
+                .setName(StreamName.format(projectId, location, streamId))
+                .setState(state);
+
+        FieldMask.Builder fieldMaskBuilder = FieldMask.newBuilder().addPaths(FIELD_STATE);
+
+        UpdateStreamRequest request = UpdateStreamRequest.newBuilder()
+                .setStream(streamBuilder)
+                .setUpdateMask(fieldMaskBuilder)
+                .build();
+
+        return datastreamClient.updateStreamAsync(request).get();
+    }
+
+    @Override
+    public Stream startStream(String streamId) throws ExecutionException, InterruptedException {
+        return updateStreamState(streamId, Stream.State.RUNNING);
+    }
+
+    @Override
+    public Stream pauseStream(String streamID) throws ExecutionException, InterruptedException {
+        return updateStreamState(streamID, Stream.State.PAUSED);
+    }
+
+    @Override
+    public void cleanUpAll() throws ExecutionException, InterruptedException {
```

Comment:
This should be named `cleanupAll()` after the inherited method in `ResourceManager.java`. 
```suggestion
    public void cleanupAll() throws ExecutionException, InterruptedException {
```